README.md

```diff
@@ -236,7 +236,7 @@ Hurray! You successfully made a contribution!
   - [Evandro Rodrigues](https://github.com/evnrodr)
 
 - ### **F**
-  
+
```

Comment:
Please can you remove this line @136tejas

Reply:
which line?
line number?

Reply:
239 please sorry for not clarifying.

Reply:
Actually I don’t think I am right. I just saw you had removed the line and then added it on. So that’s fine. Sorry @136tejas. There is nothing you need to change LGTM.